File: mssqlConnect.py
# PYTHON CONNECT TO MSSQL

import pyodbc, pandas

try:
    conn = pyodbc.connect("DRIVER={SQL Server};" "SERVER=;" "DATABASE=;" "UID=;" "PWD=;")

    sql = """
    select * from Autorzy
    """
    data = pandas.read_sql_query(sql, conn)

    conn.close()

    print(type(data))
    print(pandas.DataFrame.head(data))

except:
    print("Failed to connect database or execute query")

File: mssqlConnect.js
// NODE.JS CONNECT TO MSSQL

const sql = require("mssql");

const config = {
  server: "",
  database: "",
  user: "",
  password: "",
  options: {
    enableArithAbort: true,
  },
};

function connect() {
  const conn = new sql.ConnectionPool(config);
  const req = new sql.Request(conn);

  const query = `
  select * from dbo.Autorzy
  `;

  conn.connect((err) => {
    if (err) throw err;
    req.query(query, (err, res) => {
      if (err) throw err;
      else console.log(res);
      conn.close();
    });
  });
}

connect();
